community/logging/src/test/java/org/neo4j/logging/RotatingFileOutputStreamSupplierTest.java

```diff
@@ -314,8 +319,9 @@ public void rotationCompleted( OutputStream out )
             }
         } );
 
+        ExecutorService rotationExecutor = Executors.newSingleThreadExecutor();
```

Comment:
To make this even more correct what about wrapping it into `try...finally` block?